Resources/Prototypes/SimpleStation14/Recipes/Reactions/drink.yml

```
@@ -0,0 +1,9 @@
+- type: reaction
+  id: ChocolateMilk
+  reactants:
+    Milk:
+      amount: 4
```

```suggestion
      amount: 5
```
I think it'd be good if it was 1:1 for the milk, so you're not synthesizing milk using chocolate.